mock-guide/english/php/functions/files/file-writing/index.md

```diff
@@ -1,52 +0,0 @@
----
-title: File Writing
```

Comment:
@RandellDawson Do not delete this file. This one is used in tests.

Reply:
Ah, now I understand why it is failing.  I could not figure out for the life of me, what the issue was.

Reply:
What is the easiest way for me to just add the one file back?

Reply:
I figured it out.